torch/testing/_internal/common_methods_invocations.py

```diff
@@ -358,6 +358,20 @@ def sample_inputs_tensor_split(op_info, device, dtype, requires_grad):
                         args=(torch.tensor([1, 2, 3]),),
                         kwargs=dict(dim=1)),)
 
+def sample_inputs_tensor_hsplit_vsplit_dsplit(op_info, device, dtype, requires_grad):
+    return (SampleInput(make_tensor((S, S, S), device, dtype,
+                                    low=None, high=None,
+                                    requires_grad=requires_grad),
+                        args=(torch.tensor([1, 2, 3]),),),
```

Comment:
The current signatures only support a single scalar integer or a Python list of integers, not a tensor of integers